integreat_cms/core/settings.py

```diff
@@ -888,6 +883,16 @@
 )
 
 
+#################
+# INTERNAL URLS #
+#################
+
+#: The URLs which are treated as internal in TinyMCE custom link plugin
+INTERNAL_URLS = [BASE_URL, WEBAPP_URL, WEBSITE_URL] + [
+    x.strip(x) for x in os.environ.get("INTEGREAT_CMS_INTERNAL_URLS", "").splitlines()
+]
```

Comment:
```suggestion
INTERNAL_URLS = (
    ALLOWED_HOSTS
    + [WEBAPP_URL, WEBSITE_URL]
    + [
        x.strip(x)
        for x in os.environ.get("INTEGREAT_CMS_INTERNAL_URLS", "").splitlines()
    ]
)
```
Oh, I just noticed we sometimes have [multiple CMS domains](https://git.tuerantuer.org/DF/salt/src/commit/80f38189a906116872590cdc733683e47e95d913/pillars/letsencrypt/org/tuerantuer/malte-prod.sls#L4-L6) we add to the allowed hosts, probably all of them should be implicitly be marked as internal and `BASE_URL` isn't enough...